ephypype/commands/tests/test_cli.py

```diff
@@ -4,8 +4,8 @@
 #
 # License: BSD (3-clause)
 
-import matplotlib # noqa
-matplotlib.use('Agg')  # noqa; for testing don't use X server
+# import matplotlib # noqa
+# matplotlib.use('Agg')  # noqa; for testing don't use X server
```

Comment:
you will still need this I think, but it doesn't need to be at the top of the file before all imports ...

Reply:
I just want to reproduce the error with matplotlib backend for now because I failed to reproduce it locally. It's not the final version.

Reply:
you can't reproduce it locally because you are not running in headless mode. Try running it over `ssh` (without -X) or somewhere you can't have access to the screen

Reply:
I've been running tests without starting the X-server in hope that it will give me just that, but it didn't work

Reply:
Ok, the tests went through just because they work this way.

Reply:
okay so the tests don't cover this function probably ...

Reply:
we can remove these two lines then and I'll merge.

Reply:
No, it's just there was only one matplotlib import in global scope in a completely different place than that where the error happened before, so I didn't believe that moving the import inside fixed it. I didn't test the initial configuration.

Reply:
Also I think we should move these two lines down in every test, so Travis detects if somebody adds global pyplot import

Reply:
yes +1 for moving the two lines down in every test

Reply:
I reproduced the initial error with no X-server reverting the import changes, so it all makes sense now.